docs/data/base/guides/working-with-tailwind-css/working-with-tailwind-css.md

```diff
@@ -305,7 +308,7 @@ Let's add the `Slider` into the `Player` component now:
 
 You should see this:
 
-<img src="/static/images/base-with-tailwind-css/player-slider.png" alt="Screenshot of the media player used as example in the guide, designed by the Tailwind Labs team" style="width: 796px; margin-top: 8px; margin-bottom: 8px;" />
+<img src="/static/base/with-tailwind-css/player-slider.png" alt="Screenshot of the media player used as example in the guide, designed by the Tailwind Labs team" style="width: 796px; margin-top: 8px; margin-bottom: 8px;" />
```

Comment:
I have tried to play more with the image, I couldn't find a lot of room for improvement only to set the image width to match 

```suggestion
<img src="/static/base/with-tailwind-css/player-slider.png" alt="Screenshot of the media player used as example in the guide, designed by the Tailwind Labs team" style="width: 745px; margin-top: 8px; margin-bottom: 8px;" />
```

with the width on Figma https://www.figma.com/file/vOx7rzFQZ9e54W81NReDdU/Assets?node-id=1725%3A3280.

### Before

<img width="642" alt="Screenshot 2022-07-28 at 00 34 20" src="https://user-images.githubusercontent.com/3165635/181384106-5070b204-1640-4cb2-ae06-7af208fdee7c.png">


### After

<img width="634" alt="Screenshot 2022-07-28 at 00 34 13" src="https://user-images.githubusercontent.com/3165635/181384115-1d2c5e58-b096-42f6-a5b5-1e47098af0b6.png">

The icons seem to render more cleanly, or compare the underline under Ep.

---

cc @danilo-leal What do you think about making the framer's width on Figma always equal to the render width on the webpage as a "best practice"? Maybe using 745px as the baseline?
For the blog, the max width on desktop is 676px, on tablet it's above: 796px. I wonder if the layout on tablet isn't buggy, should we expect the same as on desktop: 676px?

Reply:
@mnajdova I have unresolved this thread, in my comparison, the screenshots are different, e.g. the sliders' thumb, I have have exported the image from the Figma frame that is linked.

I'm also eager to discuss a potential fix one layer down the stack with Danilo as I suspect we might be on the root cause of couple of preivous pain points with the images that are displayed on the blog and docs in the past. For example, we talked about how small the font size was in the past, which seems to have this same origin.

Reply:
So, should I make these images exactly at `745px` or something else? I remember you mentioning at some point to do them as 16:9?! I'm happy to tweak them so it's of better quality. Also, should I export them as 1x or 2x?

Reply:
My proposal would be to:

- For the docs pages, work in Figma with a frame that is 745px wide. Why? We set a max-width on the container to be 814.33 pixels. On a MacBook Air M2 of 13", the width of the container is 765px. Why? 745px seems to be small enough to fit inside the container down to relatively small screen sizes, which avoid any resizing, hence any pixels distortions
- For the blog pages, work in Figma with a frame that is 692px wide. Why? It's currently 676px wide but isn't this too narrow? Medium uses 692px wide so I would propose we do the same.
- Fix the CSS breakpoint on the blog so that the max width doesn't go above 692px wide on tablette. Why? I believe it grows too much, making it harder to read. The best is to experience it to judge.
- For the blog and the docs pages, to always export assets x2. Why? Because maybe devices have a device-pixel-ratio of 2 https://www.notion.so/mui-org/Analytics-f2e2576203564e6c8dc3429cf305a36f#a0ba07e675514676bca31b2e88d392f1 and the trend is going up
- Document the above somewhere, maybe on Notion on a page that covers Figma. Why? To share the knowledge.

Reply:
> Adding images to the website requires some effort (manually reducing file size, light & dark mode), this work should be delegated to headless CMS.

I think that this is a bit different root problem, but related. "reducing file size" we would need a service like https://imgix.com/, https://cloudinary.com/, https://www.fastly.com/io/, to consider the cost, it could be done as an isolated effort. "light & dark mode" not sure if we can do something to fix this one, other than only having light mode 😁. Other pains point for me would be:

- delivery encoding: we use jpg but WebP is a lot more efficient, it is some of the above make it possible to detect the supported encoding and deliver the right asset

Reply:
I have created these image guidelines for the blog https://www.notion.so/mui-org/Blog-247ec2bff5fa46e799ef06a693c94917#af943b7359c7439d8457dee53f51f184 and https://www.notion.so/mui-org/Documentation-5a1686758f4f4267989b67a1c2c270da.